src/style.css

```diff
@@ -1 +1,62 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  color: rgba(255, 255, 255, 1);
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.image__area {
+  margin: 10px 0 0 50px;
+  flex-grow: 1;
+}
+
+.icon {
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+}
+
+.list {
+  width: 808px;
+  margin: 0;
+  padding: 0 20px 0 0;
```

Comment:
Let's simplify it
```suggestion
  padding-right: 20px;
```

Reply:
If I write like that, left padding will be 40px (in default), so I must install padding 0